subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/AbstractConsoleBuildResultFunctionalTest.groovy

```diff
@@ -112,7 +114,7 @@ BUILD SUCCESSFUL in [ \\dms]+
 
         and:
         // Ensure the failure is a location that the fixtures can see
-        failure.assertHasDescription("Execution failed for task ':broken'")
+        failure.assertThatDescription(containsString("Execution failed for task ':broken'"))
```

Comment:
🤔 I think it makes sense to use `contains` only for the debug log level. For the other log levels, nothing should slip into to description. WDYT?